addon/globalPlugins/EnhancedDictionaries/guiHelper.py

```diff
@@ -181,11 +192,17 @@ def hasEntry(self, pattern):
 	def onOk(self, evt):
 		newDictionary = not os.path.exists(self.speechDict.fileName)
 		super().onOk(evt)
+
+		profileName = config.conf.getActiveProfile().name or "normal configuration"
+		if profileName != "normal configuration":
+			checkboxValue = self.keepUpdatedCheckBox.GetValue()
+			profileConfigurationHelper.saveCheckboxValueForProfile(checkboxValue)
```

Comment:
Variable name suggestion
```suggestion
			profileConfigurationHelper.saveKeepDictionaryUpdatedCheckboxValueForProfile(checkboxValue)
```